work-process/user-stories.md

```diff
@@ -9,6 +9,14 @@
   *  they assign people from whom they want to receive feedback for better definition of the User Story.
   * Once these people post their feedback, they unassign themselves from the User Story.
 - Once the Scrum Master and Product Owner agree that the User Story is well defined, they add ``✋ [vote]`` prefix to the title of the User Story. At that point, Scrum Master and/or Product Owner assign people from whom they want to receive User Story Points estimation. With this, online poker planning begins.
+-  Assignees add their estimates in a comment with the following structure:
+```
+<details>
+  <summary>Click to see estimate</summary>
+  SP: (1, 2, 3, 5, 8, 13, 20, 100)
```

Comment:
My proposal

First is to not have the values hard-coded in the handbook and simply inform where to look (issue estimate drop-down).
Second using `click to see` begs you to click it when scrolling down the pages so instead replace it with something more benign:
```
<details>
  <summary>Story Estimate</summary>
  SP: <value>
</details>
```